.github/workflows/test.yml

```diff
@@ -0,0 +1,21 @@
+name: Test
```

Comment:
Was about to ask how we'll handle linting without Travis, but you're way ahead of me!